src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java

```diff
@@ -52,12 +52,21 @@ public Command feedToShooter() {
         return Commands.runOnce(() -> io.setFeederVelocity(1200), this);
     }
 
-    // TODO: Add method to check if feeder is spun up
+    public boolean isFeederAtSetpoint() {
+        return Math.abs(inputs.feederVelocityError) < VELOCITY_ERR_TOLERANCE_RPM;
+//        return MathUtil.isNear(inputs.setpointRPM, inputs.feederVelocityRPM, VELOCITY_ERR_TOLERANCE_RPM);
+    }
 
     public void stopFeeder() {
-        io.setFeederVoltage(0);
+        io.setFeederVoltage(0); // shouldn't stopFeeder probably set its velocity?
```

Comment:
I don't want it to actively move towards zero velocity, I want it to stop trying to move, brake mode will take care of the rest